yt/frontends/ramses/data_structures.py

```diff
@@ -630,7 +635,13 @@ def read_rhs(cast):
         # This is likely not true, but it's not clear how to determine the boundary conditions
         self.periodicity = (True, True, True)
         # These conditions seem to always be true for non-cosmological datasets
-        if rheader["time"] >= 0 and rheader["H0"] == 1 and rheader["aexp"] == 1:
+        if self.force_cosmological is not None:
+            is_cosmological = self.force_cosmological
+        else:
+            is_cosmological = (rheader["time"] >= 0 and
+                               rheader["H0"] == 1 and
+                               rheader["aexp"] == 1)
```

Comment:
It looks like the fix is helping to pass the test with parallelism, but I still don't understand why they were passing in serial.